### PR TITLE
Adds tekton pipelines v0.8.0 release config

### PR DESCRIPTION
**Pull request by hrishin**
- Adds ci-operator config and prow jobs(pre and post submit)
- Adds image mirroring

cc: @chmouel @vdemeester  @sthaha

### CODE REVIEW

File: ci-operator/config/openshift/tektoncd-pipeline/openshift-tektoncd-pipeline-release-v0.8.0.yaml [hunk 1]
```diff
@@ -0,0 +1,125 @@
+tag_specification:
+  name: '4.2'
+  namespace: ocp
```
Comment (hrishin) [inline ci-operator/config/openshift/tektoncd-pipeline/openshift-tektoncd-pipeline-release-v0.8.0.yaml]: Changed to OpenShift 4.2

Comment (chmouel) [reply ci-operator/config/openshift/tektoncd-pipeline/openshift-tektoncd-pipeline-release-v0.8.0.yaml]: we need to do this for master!

**Comment by chmouel** [comment]
/hold

@hrishin can you add a file to tag the openshift-catalog image  as described here? https://hackmd.io/C0Ayxi4gR1aXSE3Nzj_XZA?edit

**Comment by hrishin** [comment]
@chmouel oh yes, you mean https://github.com/openshift/release/pull/5590/files#diff-fefcb9e4ca837979254779110662bb16R1-R12 ? :thinking:

**Comment by chmouel** [comment]
/hold cancel
/lgtm
/approve
/meow

Thanks 🙇🏽‍♂️

**Pull request merged**